src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PersonList } from '../components/PersonList';
+import { ErrorMessage } from '../components/ErrorMessage';
+import { NoPeopleMessage } from '../components/NoPeopleMessage';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [errorMessage, setErrorMassage] = useState(false);
```

Comment:
There is a typo in the state variable name 'setErrorMassage'. It should be 'setErrorMessage' to match the purpose of the variable.